README.md

```diff
@@ -1,32 +1,79 @@
-# Composition vs Inheritance
+# Composition vs. Inheritance
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Explain the difference between composition and inheritance
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Composition and inheritance represent two types of relationships
+between classes. Composition and inheritance both promote code reuse,
+but in different ways.
 
-## Topic 1
+## When to use Composition and Inheritance
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+**Composition** is when a class has references to other classes as members.
+We can think of composition as having a "has-a" relationship:
 
-## Topic 2
+- A library has a book.
+- A house has a kitchen.
+- A car has a steering wheel
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+Consider the following example:
 
-## Conclusion
+```java
+public class SteeringWheel {
+    private double radius;
+    private boolean heated;
+    ...
+}
 
-A short one or two paragraph summary of the contents of the lessons, recapping
-the learning goals.
+public class Engine {
+    private String type;
+    private int cylinders;
+    ...
+}
 
-## Resources
+public class Car {
+    private SteeringWheel steeringWheel;
+    private Engine engine;
+    ...
+}
+```
 
-- [Resource Link 1](example.com)
-- [Resource Link 2](example.com)
+This is a very simplified example, but it shows that we know a car has as
+parts a steering wheel, an engine, etc. So we can represent this in code
+by defining the `Car` class as having fields to reference 
+`SteeringWheel` and `Engine` objects.
```

Comment:
Re-word suggestion: "We can represent this in code by defining a `Car` class to have fields referencing a `SteeringWheel` and an `Engine` object."

Reply:
reworded.